linera-witty/src/runtime/wasmer/mod.rs

```diff
@@ -213,6 +226,13 @@ impl<UserData> InstanceSlot<UserData> {
             .cloned()
     }
 
+    /// Returns a reference to the [`wasmer::Instance`] stored in this [`InstanceSlot`].
```

Comment:
Maybe add that it panics if it's already locked.

Reply:
Good point! I also made it `&mut self` to prevent it being shared and leading to the panic :sweat_smile: